scripts/release-tools/downstream/y-stream-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -eoux pipefail
+
+previous_release=$1
+release=$2
```

Comment:
```suggestion
previous_release="$1"
release="$2"
```

Reply:
Would be also good to validate the inputs and stop the script early with a help message if the inputs make no sense.